cmd/crictl/config.go

```diff
@@ -106,7 +106,7 @@ CRICTL OPTIONS:
 			}
 			return common.WriteConfig(config, configFile)
 		} else if c.Bool("list") {
-			display := newTableDisplay(20, 1, 3, ' ', 0)
```

Comment:
I wonder what linter rule did it break?

Reply:
It's the [gosimple](https://golangci-lint.run/usage/linters/#gosimple) linter, but I could work around by introducing a `newDefaultTableDisplay` method and reusing it everywhere where it's needed.